Source/Core/DolphinQt/Settings/InterfacePane.cpp

```diff
@@ -372,6 +382,10 @@ void InterfacePane::AddDescriptions()
       QT_TR_NOOP("Shows on-screen display messages over the render window. These messages "
                  "disappear after several seconds."
                  "<br><br><dolphin_emphasis>If unsure, leave this checked.</dolphin_emphasis>");
+  static const char TR_OSD_FONT_SIZE_DESCRIPTION[] = QT_TR_NOOP(
+      "Changes the font size of the On Screen Display. Affects features such as performance "
+      "statistics, frame counter, and netplay chat.<br><br><dolphin_emphasis>If "
+      "unsure, leave this at 14.</dolphin_emphasis>");
```

Comment:
Let's avoid saying the default value here.  This is a great example of why...as the default is actually 13.

Reply:
What should it say?  Should it just not recommend a value?

Alternatively I could change it from a font point value to a scale 100% - 300%.